### PR TITLE
Closes AgileVentures/MetPlus_tracker#228

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: app/models/job_seeker.rb [hunk 1]
```diff
@@ -24,13 +24,29 @@ def is_job_seeker?
...
   def self.your_jobseekers_jd(job_developer)
     # this method serves the Job Developer home page, hence the "your"
-    where(:id => AgencyRelation.in_role_of(:JD).where(:agency_person => job_developer).pluck(:job_seeker_id)).order("users.last_name")
+
+    where(id: self.with_ap_in_role(:JD, job_developer)).
+            includes(:job_seeker_status, :job_applications).
+            order("users.last_name")
+  end
+
+  def self.with_ap_in_role(role_key, agency_person)
```
Comment (joaopapereira) [inline app/models/job_seeker.rb]: Do we have tests for this function?

**Comment by joaopapereira** [comment]
Looks good, please review my comment

File: app/models/job_seeker.rb (hunk 1 above)
Comment (patmbolger) [inline app/models/job_seeker.rb]: Will add that.

**Comment by joaopapereira** [comment]
Looks good now, going to merge

**Pull request merged**